<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="LAPTOP-7TGI8T0C" failures="1" tests="1" name="com.runner.TestRunner" time="12.110" errors="0" timestamp="22 Mar 2023 05:48:58 GMT" skipped="0">
  <testcase classname="com.runner.TestRunner" name="runScenario" time="12.110">
    <failure message="expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertTrue(Assert.java:44)
at org.testng.Assert.assertTrue(Assert.java:54)
at com.steps.StepDefinition.user_print_page_title(StepDefinition.java:56)
at âœ½.User print page title(file:src/test/resources/features/loginPage.feature:9)
]]>
    </failure>
  </testcase> <!-- runScenario -->
</testsuite> <!-- com.runner.TestRunner -->
